### PR TITLE
21 add file

### PR DESCRIPTION
**Pull request by enleiwu**
navigation/Stacks.tsx
- Typescript interface schema added to remove errors in line 59
- List navigator screen added

Home.tsx
- Added navigation action on list click with list data

List
- Created component and styling

Incomplete
- Theme color from list data missing

### CODE REVIEW

File: src/resources/strings.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ const strings = new LocalizedStrings({
   en: {
     add_icon: 'Add Icon',
     add_list_name: 'Add List Name',
+    add_file: 'Add New File',
```
Comment (jentoobento) [inline src/resources/strings.js]: nitpicking, but can we keep this file alphabetical? add comment at top for future reference ?

Comment (enleiwu) [reply src/resources/strings.js]: Does 'f' not come after 'i' and 'l' in the alphabet?

File: src/screens/Home/Home.tsx [hunk 2]
```diff
@@ -94,8 +94,18 @@ const Home = () => {
...
   };
 
-  const renderLists = ({iconName, iconType, id}) => (
-    <Pressable key={id} style={styles.button}>
+  const renderLists = ({iconName, iconType, id, name}) => (
+    <Pressable
+      key={id}
+      style={styles.button}
+      onPress={() =>
+        navigation.navigate('List', {
```
Comment (jentoobento) [inline src/screens/Home/Home.tsx:102]: Depending on how we structure the data I'm not sure passing the whole list item as a prop to the route is a good idea. Such as if a list looks like: { id: 1, name: 'house', files: [ {id: 1}, {id: 2}, ...{id: 999}] }

File: src/screens/Home/Home.tsx [hunk 3]
```diff
@@ -94,8 +94,18 @@ const Home = () => {
...
+          id,
```
Comment (jentoobento) [inline src/screens/Home/Home.tsx:103]: Possibly passing just the id would be cleaner and grabbing the corresponding list by id in the List screen

**Review by jentoobento — APPROVED** [review]

**Pull request merged**